### DISCUSSION TITLE
Render html inside Stack()

### DISCUSSION THREAD

**Discussion by gfspiteri** [Appsilon/shiny.fluent#171] [Q&A] [answered]
I need to render html code inside a Stack() list. I have been trying in different ways using different combinations of HTML() or div() inside Stack() but I cannot get it to work. Any tips? 
Thanks!

**Comment by jakubsob** [comment, marked as answer]
Hi @gfspiteri,

this is not supported yet by shiny.react. Please look [at this issue](https://github.com/Appsilon/shiny.react/issues/13) and the suggested workaround.
